### ISSUE TITLE
Adding Micro.blog to Navigation

### ISSUE THREAD

**Issue by jbpyn** [nielsenramon/chalk#146]
I'm intending to use Chalk for [micro.blog](https://micro.blog/), but it appears adding it to the social links for navigation is a bit beyond me at the moment. Chalk seems like the ideal template for micro.blog users. 

I'm happy to muddle through, if somebody can point me in the right direction?

Thanks again

**Comment by migueldemoura** [comment]
To add a new icon to the navigation:

- Add the social network's svg icon here: https://github.com/nielsenramon/chalk/tree/master/_assets/icons (make sure it follows the same structure as the other ones)
- Add a new entry here: https://github.com/nielsenramon/chalk/tree/master/_includes/navigation.html
(just copy one of the existing ones and modify accordingly)
- Add a new social entry here: https://github.com/nielsenramon/chalk/blob/master/_config.yml#L18
(use the same flag name as in `navigation.html` file)

Hope that helps! Let me know if something's not clear enough. :smile:

**Comment by jbpyn** [comment]
Thanks again, this is great. It's nice to know I was on the right path. :)

**Comment by gammacray** [comment]
Couple years later, I'm having the same issues. I've followed @migueldemoura 's instructions (and I was doing this before I came here looking for advice) and I can get the link to show up in the header, but the icon is blank. I've named the icon, I've pointed the nav.html and config.yml at it, etc. I'm having a hard time understanding what I'm doing wrong.

For reference, I've taken an existing entry 'docker' for instance, added my own icon in it's path by renaming 'myicon.svg' to 'docker.svg' and my new icon doesn't show up there either... it's just blank. The link is there but the icon doesn't show up.

I must be missing something ridiculously simple.
